hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/contracts/HederaEvmWorldUpdater.java

```diff
@@ -32,4 +42,8 @@ public interface HederaEvmWorldUpdater extends WorldUpdater {
      * @return the amount of Gas to refund;
      */
     long getSbhRefund();
+
+    Address priorityAddress(final Address addressOrAlias);
+
+    Address newAliasedContractAddress(final Address sponsor, final Address alias);
```

Comment:
I believe we can just delete these 2 methods. We would just implement them in the corresponding copied `HederaStackedWorldStateUpdater` logic inside `mirror-node`.